server/model/taggedBailEditModel.ts

```diff
@@ -57,7 +57,7 @@ export default class TaggedBailEditModel {
   }
 
   private getCaseDetailsCell(): string {
-    return `${this.getCourtName()} ${isSentenceRecalled(this.sentenceAndOffence.sentenceCalculationType) ? getSentenceRecallTagHTML() : ''}<br>${this.getCaseReference()} ${this.getSentenceDate()}`
+    return `${this.getCourtName()} | ${this.getCaseReference()} ${isSentenceRecalled(this.sentenceAndOffence.sentenceCalculationType) ? getSentenceRecallTagHTML() : ''}<br>${this.getSentenceDate()}`
```

Comment:
Should this be a `|` character? I think these things are normally CSS lines otherwise screenreaders get confused.

Reply:
(applies to everywhere in this PR.)

Reply:
I've replaced this with CSS